File: src/main/java/com/pcb/pcbridge/bukkit/players/PlayerData.java
package com.pcb.pcbridge.bukkit.players;

public final class PlayerData 
{
	public long UserId;
	public boolean SwearFilter;
	
	public PlayerData() { }
	public PlayerData(long userId, boolean swearFilter)
	{
		this.UserId = userId;
		this.SwearFilter = swearFilter;
	}
}

File: pcbridge-http/src/main/kotlin/com/projectcitybuild/pcbridge/http/models/pcb/PlayerData.kt
package com.projectcitybuild.pcbridge.http.models.pcb

import com.google.gson.annotations.SerializedName
import kotlinx.serialization.Serializable

@Serializable
data class PlayerData(
    @SerializedName("account")
    val account: Account? = null,

    @SerializedName("player")
    val player: Player? = null,

    @SerializedName("groups")
    val groups: List<Group> = emptyList(),

    @SerializedName("ban")
    val playerBan: PlayerBan? = null,

    @SerializedName("ip_ban")
    val ipBan: IPBan? = null,

    @SerializedName("badges")
    val badges: List<Badge> = emptyList(),
)
